### PR TITLE
Set m_TargetCorners in TrackedTarget constructor

### PR DESCRIPTION
**Pull request by OfriRosenbaum**

### CODE REVIEW

File: photon-core/src/main/java/org/photonvision/vision/target/TrackedTarget.java [hunk 1]
```diff
@@ -66,6 +66,8 @@ public TrackedTarget(
         this.m_mainContour = origTarget.m_mainContour;
         this.m_subContours = origTarget.m_subContours;
         this.m_shape = shape;
+        Point[] corners = shape.contour.getConvexHull().toArray();
+        m_targetCorners = List.of(corners);
```
Comment (mcm001) [inline photon-core/src/main/java/org/photonvision/vision/target/TrackedTarget.java:70]: I'm like pretty sure the order that this happens in is OK. Can you confirm that doing this won't overwrite corners detected by like cornerdetectionpipe? Glancing thru the code this looks fine

Comment (OfriRosenbaum) [reply photon-core/src/main/java/org/photonvision/vision/target/TrackedTarget.java:70]: Unfortunately I didn't manage to simulate the changes on my rpi/computer

**Comment by Lewis-Seiden** [comment]
Briefly tested this on a robot today, it seems to work. I'll get some more time to test it tomorrow.

**Comment by Lewis-Seiden** [comment]
Got to test it a little more running triangle shape detection, and am getting back 8 vertices instead of the 3 I expected.

**Comment by OfriRosenbaum** [comment]
> Got to test it a little more running triangle shape detection, and am getting back 8 vertices instead of the 3 I expected.
@Lewis-Seiden 
Interesting. Got any idea which are the real 3 vertices?

**Comment by Lewis-Seiden** [comment]
> > Got to test it a little more running triangle shape detection, and am getting back 8 vertices instead of the 3 I expected.
> 
> @Lewis-Seiden Interesting. Got any idea which are the real 3 vertices?

I think they are the vertices before shape simplification, since the vertices are around the corners of the triangle
image

**Comment by mdurrani808** [comment]
Any updates on this?

**Comment by OfriRosenbaum** [comment]
> Any updates on this?

Not as far as I know of. From my understanding if we can get the vertices **after** the shape simplification, it will all work well. Unfortunately I don't know how to do it so I would be glad if someone that actually knows it were doing it instead

**Pull request closed without merging**